### PR TITLE
Add configs, `init` command, and `keys` command

### PR DESCRIPTION
**Pull request by evan-forbes**
Add configs and two commands. The `init` commands creates the .dalc repo and saves default configs, while the `keys` command is imported from `cosmoscmd` to add celestia key management from the cli.

Again, apologies for code that is not the most polished. @tzdybal and I are sprinting to get a not-ideal-but-definitely-working dalc done by devnet.

blocked by #11

### CODE REVIEW

File: cmd/dalc/main.go [hunk 1]
```diff
@@ -3,22 +3,63 @@ package main
...
+	"github.com/tendermint/spm/cosmoscmd"
 )
 
 func main() {
 	root := rootCmd()
+
+	cosmoscmd.SetPrefixes(app.AccountAddressPrefix)
+
+	root.AddCommand(
+		keys.Commands(config.DefaultConfigPath),
```
Comment (evan-forbes) [inline cmd/dalc/main.go:19]: this is all it takes to add the `keys` command! now we can manage keys via the cli. obvi not suitable for mainnet, but definitely workable for devnet

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	rawCfg, err := os.ReadFile(path)
+	if err != nil {
+		return cfg, err
+	}
+	_, err = toml.Decode(string(rawCfg), &cfg)
+	if err != nil {
+		return cfg, err
+	}
+	return cfg, nil
+}
```
Comment (evan-forbes) [inline config/config.go:57]: I didn't write tests for these, hope that alright considering they're super basic

File: config/config.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	return ServerConfig{
+		// todo(evan): add default configs later
+		// BlockSubmitterConfig: ,
+		// LightClientConfig: ,
+		KeyringConfig: DefaultKeyringConfig(),
+	}
+}
+
+type BlockSubmitterConfig struct {
+}
```
Comment (evan-forbes) [inline config/config.go:69]: Will fill this out in #12

File: cmd/dalc/main.go [hunk 4]
```diff
@@ -3,22 +3,63 @@ package main
...
+		initCmd(),
+	)
 	if err := root.Execute(); err != nil {
 		os.Exit(1)
```
Comment (tzdybal) [inline cmd/dalc/main.go:23]:
```suggestion
		log.Fatal(err)
```

File: cmd/dalc/main.go [hunk 5]
```diff
@@ -3,22 +3,63 @@ package main
 import (
 	"os"
```
Comment (tzdybal) [inline cmd/dalc/main.go:4]:
```suggestion
	"log"
	"os"
```

File: cmd/dalc/main.go [hunk 6]
```diff
@@ -3,22 +3,63 @@ package main
 import (
```
Comment (tzdybal) [inline cmd/dalc/main.go:3]:
```suggestion
import (
	"log"
```

File: cmd/dalc/main.go (hunk 5 above)
Comment (tzdybal) [inline cmd/dalc/main.go:4]:
```suggestion
import (
	"log"
	"os"
```

File: config/config.go (hunk 2 above)
Comment (tzdybal) [reply config/config.go:57]: I created (good-first-)issue for this: https://github.com/celestiaorg/dalc/issues/14

**Review by tzdybal — APPROVED** [review]
LGTM. Let's not block further work.

**Comment by adlerjohn** [comment]
Are you going to rebase against `main` @evan-forbes?

**Pull request merged**